packages/components/pie-button/tsup.config.ts

```diff
@@ -0,0 +1,16 @@
+import type { Options } from 'tsup';
```

Comment:
It might be worth pulling out into a shared config but we could do this when we add the next component

Reply:
I would def do this when we come to creating the next component – we can do this alongside the more general TS config